torch/csrc/jit/passes/onnx/peephole.cpp

```diff
@@ -623,6 +623,45 @@ static void eraseListConstruct(Block* block, int opset_version) {
   eraseListConstruct(block->return_node(), opset_version);
 }
 
+static void eraseListUnpack(Block* block, int opset_version);
```

Comment:
Why we cannot do a symbolic for `prim::ListUnpack`?

Reply:
Because this op (and `prim::ListUnpack`) is converted based on patterns involving other operators. Also in symbolic functions, many operator create these two as well.